src/Event/PostIndexPopulateEvent.php

```diff
@@ -11,6 +11,6 @@
 
 namespace FOS\ElasticaBundle\Event;
 
-class PostIndexPopulateEvent extends IndexPopulateEvent
+final class PostIndexPopulateEvent extends IndexPopulateEvent
```

Comment:
Maybe we should also mark the corresponding base class as `abstract`?

Reply:
done

Reply:
Hm tests fail if I add abstract. Should the tests be changed or should we refrain from adding abstract? 

https://travis-ci.com/github/FriendsOfSymfony/FOSElasticaBundle/jobs/451397287

Reply:
An abstract class should not be initialized.

If the class was not meant to be abstract, removing the keyword should be enough.

Reply:
Ok, then its done. The result is no abstract classes.